src/js/app.js

```diff
@@ -1,3 +1,43 @@
+import Headroom from '../js/lib/headroom.min.js'
+
+function show () {
+  $('.mobile-nav').addClass('is-active')
+  $('.hamburger-button').css('display', 'none')
+  $('.mobile-nav').animate({opacity: 1}, 200)
+  $('html').css('overflow', 'hidden')
+}
+
+function hide () {
```

Comment:
show() and hide() should be renamed to something more descriptive since it seems like they're only usable for nav purposes